### PR TITLE
First pass intg CAMO limitations with limitations

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

First pass integrating CAMO limitations with master PGD limitations page.

## Background

Requested to bring more limitations into the limitations page to centralise limitations. Back links into anchored sections in limitations now allow other docs to refer into limitations where they are specific to that subject matter.

### CODE REVIEW

File: product_docs/docs/pgd/5/limitations.mdx [hunk 1]
```diff
@@ -2,47 +2,130 @@
...
+This section covers design limitations of EDB Postgres Distributed (PGD), that
+should be taken into account when planning your deployment.
 
-## Limits on nodes
+## Limitations related to the number of nodes
 
-- PGD can run hundreds of nodes assuming adequate hardware and network. However, for mesh-based deployments, we generally don’t recommend running more than 48 nodes in one cluster. If extra read scalability is needed beyond the 48 node limit, subscriber only nodes can be added without adding connections to the mesh network.
+- PGD can run hundreds of nodes assuming adequate hardware and network. However,
+  for mesh-based deployments, we generally don’t recommend running more than 48
+  nodes in one cluster. If extra read scalability is needed beyond the 48 node
```
Comment (drothery-edb) [inline product_docs/docs/pgd/5/limitations.mdx:12]: Can we make this sentence active? For example,
```suggestion
  nodes in one cluster. If you need extra read scalability beyond the 48 node
```

**Comment by djw-m** [comment]
Done.

**Review by drothery-edb — DISMISSED** [review]

File: product_docs/docs/pgd/5/limitations.mdx [hunk 2]
```diff
@@ -2,47 +2,131 @@
...
+as a result of a restart, there's no easy way to recover
+it. Therefore, we recommend that applications wait for outstanding
+transactions to end before shutting down.
+
+-  For the client to apply proper checks, a transaction protected by CAMO
+can't be a single statement with implicit transaction control. You also can't
+use CAMO with a transaction-controlling procedure or
+in a `DO` block that tries to start or end transactions.
+
+- CAMO resolves commit status but doesn't yet resolve pending
```
Comment (PJMODOS) [inline product_docs/docs/pgd/5/limitations.mdx:95]:
```suggestion
- CAMO resolves commit status but doesn't resolve pending
```

**Review by PJMODOS — APPROVED** [review]
👍

**Pull request merged**